### PR TITLE
Ignore power

### LINKED ISSUE #253: Option to ignore power
Running this software on a Raspberry Pi and i want the option to skip power check cause if i run it as a service it bumps it up to performance since it thinks it is a laptop so if there could be a param in  the config like 

ignore_power=yes

so it can focus on giving a low power raspberry pi the chance to go even lower in power consumption by powersaving and giving it power when its needed.

### PR DESCRIPTION
**Pull request by JakeRoggenbuck**
fixes #253

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ also the default settings if no config is provided
 # acs.toml
 powersave_under = 20
 overheat_threshold = 80
+ignore_power = false
```
Comment (Shuzhengz) [inline README.md:98]: Prob would be a good idea to give explanation of what it does (in here or in wiki)

File: src/daemon.rs [hunk 2]
```diff
@@ -192,7 +192,9 @@ impl Checker for Daemon {
     fn start_charging_rule(&mut self) -> Result<(), Error> {
         if self.charging && !self.already_charging {
             if self.lid_state == LidState::Closed || self.charge < self.config.powersave_under {
-                self.lid_closed_or_charge_under();
+                if !self.config.ignore_power {
```
Comment (Shuzhengz) [inline src/daemon.rs:195]: wait but would this ignore the thermal threshold if ignore power is true?

**Comment by JakeRoggenbuck** [comment]
Being redone in #291. The new PR will be merged soon.

**Comment by JakeRoggenbuck** [comment]
Done in #291

**Pull request closed without merging**